src/SpectrumCalculationModule.ts

```
@@ -0,0 +1,475 @@
+import { WindowType, Quantity, TimeWaveform, SpectrumResult, TrendData, CONSTANTS } from './types';
+import FFT from 'fft.js';
+
+export class SpectrumCalculationModule {
+  private numberOfLines: number = 1024;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)